### PR TITLE
SEO-2245_Site_Audit_Issues

### PR DESCRIPTION
**Pull request by gowthamanthavasiyappan**
Hi @Aishwarya-Ganesan,

Worked on Bing report issues, merge these changes.

Regards,
Gowthaman

### CODE REVIEW

File: uwp/Map/Annotations.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Annotations in UWP Map control | Syncfusion
-description: Learn here all about Annotations support in Syncfusion UWP Map (SfMaps) control and more.
+description: Learn here all about Annotations support in Syncfusion Essential UWP Map (SfMaps) control, its elements, and more.
```
Comment (Aishwarya-Ganesan) [inline uwp/Map/Annotations.md:4]: Why Essential added here?

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented

File: uwp/Map/Annotations.md (hunk 1 above)
Comment (gowthamanthavasiyappan) [reply uwp/Map/Annotations.md:4]: The description length is below 100 characters, so I added essential and its elements words.

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine

**Pull request merged**